csharp/ql/src/semmle/code/csharp/frameworks/EntityFramework.qll

```diff
@@ -107,29 +114,17 @@ module EntityFramework {
     MappedProperty() {
       this = any(MappedType t).getAMember() and
       this.isPublic() and
-      not this.getAnAttribute() instanceof DataAnnotations::NotMappedAttribute
+      not isNotMapped(this)
     }
   }
 
+  /** A flow summary for EntityFramework. */
+  abstract class EFSummarizedCallable extends SummarizedCallable { }
```

Comment:
Why do we need this empty abstract class? Is this only used for grouping together the EF related summaries in tests? It's strange to see test related classes here.

Reply:
Yes, used for testing only.